git-info-generator.mjs

```diff
@@ -0,0 +1,24 @@
+import { execSync } from 'child_process';
+import { writeFileSync } from 'fs';
+
+try {
+  execSync('git --version');
```

Comment:
Wrapping this and the rest of calls in the same try-catch can show a misleading error message. 

If git repo data isn't available it will show "git not installed' which is incorrect.